client/src/modules/Mentor/pages/Interviews/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { CoursesInterviewsApi, InterviewDto } from 'api';
+import { PageLayout } from 'components/PageLayout';
+import { useLoading } from 'components/useLoading';
+import { useMemo, useState } from 'react';
+import { useAsync } from 'react-use';
+import { CourseService, Interview } from 'services/course';
+import { CoursePageProps } from 'services/models';
+import { InteviewCard } from './components/InterviewCard';
+
+export function Interviews(props: CoursePageProps) {
+  const [interviews, setInterviews] = useState<InterviewDto[]>([]);
+  const [loading, withLoading] = useLoading();
+
+  const loadData = async () => {
+    const { data } = await new CoursesInterviewsApi().getInterviews(props.course.id);
```

Comment:
Please consider to use `gerServerSideProps`  for API calls on page load

Reply:
We faced with similar issue on mentor dashboard page. Page "blinks" when data is in loading state then is loaded. Page is "jumping". So, if you do not need to reload this data later on the page, it would be preferable to load it on server side and show it immediately when page was opened

Reply:
interesting. we have a lot of pages, where the csr is used and seems there are no such problems. For now leaving as is, since it is not finished yet. I may revisit it once the flow is completed and the next MRs